src/app/pages/categories/categories.component.html

```diff
@@ -2,7 +2,7 @@
   <div class="categories-category" *ngFor="let category of categories">
     <a class="categories-header" [id]="category.slug + '-header'" (click)="navigate(category)">
```

Comment:
Why using click function instead of routerLink ?

Reply:
Had an issue with queryParams fisrt time we tried to use it. Solved yesterday night but forgot to replicate on this file. It will be done

Reply:
That way we can pass an object which doesn't work with routerLink